engine/src/main/java/org/hibernate/validator/constraints/time/DurationMin.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Hibernate Validator, declare and validate application constraints
+ *
+ * License: Apache License, Version 2.0
+ * See the license.txt file in the root directory or <http://www.apache.org/licenses/LICENSE-2.0>.
+ */
+package org.hibernate.validator.constraints.time;
+
+import static java.lang.annotation.ElementType.ANNOTATION_TYPE;
+import static java.lang.annotation.ElementType.CONSTRUCTOR;
+import static java.lang.annotation.ElementType.FIELD;
+import static java.lang.annotation.ElementType.METHOD;
+import static java.lang.annotation.ElementType.PARAMETER;
+import static java.lang.annotation.ElementType.TYPE_USE;
+import static java.lang.annotation.RetentionPolicy.RUNTIME;
+
+import java.lang.annotation.Documented;
+import java.lang.annotation.Repeatable;
+import java.lang.annotation.Retention;
+import java.lang.annotation.Target;
+import java.time.Duration;
+import java.time.temporal.ChronoUnit;
+import javax.validation.Constraint;
+import javax.validation.Payload;
+import javax.validation.ReportAsSingleViolation;
+
+import org.hibernate.validator.constraints.time.DurationMin.List;
+
+/**
+ * Annotated {@link Duration} element's length must be greater than or equal to
+ * the one constructed from {@link DurationMin#value()} in {@link DurationMin#units()}.
+ *
+ * @author Marko Bekhta
+ * @hv.experimental
+ */
+@Documented
+@Constraint(validatedBy = { })
+@Target({ METHOD, FIELD, ANNOTATION_TYPE, CONSTRUCTOR, PARAMETER, TYPE_USE })
+@Retention(RUNTIME)
+@Repeatable(List.class)
+@ReportAsSingleViolation
+public @interface DurationMin {
+
+	String message() default "{org.hibernate.validator.constraints.time.DurationMin.message}";
+
+	Class<?>[] groups() default { };
+
+	Class<? extends Payload>[] payload() default { };
+
+	long value() default 0;
+
+	ChronoUnit units() default ChronoUnit.NANOS;
```

Comment:
Yeah, looks like you only have the `parse` method for that. Or you have to use the `plus*` methods. That being said, we would do the operation only once at startup so I don't think it's that bad.

@marko-bekhta WDYT?

As for me, I find it more readable to say 4 hours 30 minutes rather than converting everything to minutes but maybe it's just me.